app/preprints/index/styles.scss

```diff
@@ -0,0 +1,213 @@
+// stylelint-disable max-nesting-depth, selector-max-compound-selectors
+
+.preprints-page-container {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    .preprint-header {
+        border-bottom: 1px solid var(--primary-color);
+        width: 100%;
+        background-color: var(--secondary-color);
+        color: var(--primary-color) !important;
+
+        a:not(.btn) {
+            color: var(--primary-color);
+            text-decoration: underline;
+        }
+
+        .description {
+            color: var(--primary-color);
+        }
+
+        .submit-container {
+            width: 100%;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            color: var(--primary-color);
+
+            .or-container {
+                font-size: 21px;
+                margin-bottom: 20px;
+                font-weight: 300;
+                line-height: 1.4;
+                color: $color-text-white;
+        
+                width: 100%;
+                display: flex;
+                flex-direction: column;
+                justify-content: center;
+                align-items: center;
+            }
+
+            .example-container {
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                width: 100%;
+                margin-top: 25px;
+            }
+        }
+    }
+
+    /* Subject Panel */
+    .preprint-subjects-container {
+        background-color: $color-bg-gray-lighter;
+        border-bottom: 1px solid #6d8a98;
```

Comment:
We should have color variables for the colors in this and other files, rather than putting raw color codes in the style files.